ImperatorToCK3/Data_Files/configurables/nickname_map.txt

```diff
@@ -31,50 +31,50 @@ link = { imp = NICKNAME_CURIO imp = NICKNAME_THE_DEVOUT ck3 = nick_the_pious }
 link = { imp = NICKNAME_THE_PARAGON ck3 = nick_the_shepherd }
 link = { imp = NICKNAME_LUPUS imp = NICKNAME_THE_WOLF ck3 = nick_the_wolf }
 link = { imp = NICKNAME_MARIUS ck3 = nick_the_overseer }
-link = { imp = NICKNAME_BRITANNICUS imp = NICKNAME_THE_BRITON ck3 = nick_the_lionheart }
-#link = { imp = NICKNAME_GALLICUS imp = NICKNAME_THE_GAUL ck3 = ??? } #"The French"
-#link = { imp = NICKNAME_BELGICUS imp = NICKNAME_THE_BELGAE ck3 = ??? } #"The Belgian"
+link = { imp = NICKNAME_BRITANNICUS imp = NICKNAME_THE_BRITON ck3 = nick_the_british }
+link = { imp = NICKNAME_GALLICUS imp = NICKNAME_THE_GAUL ck3 = nick_the_french }
+link = { imp = NICKNAME_BELGICUS imp = NICKNAME_THE_BELGAE ck3 = nick_the_belgian }
 link = { imp = NICKNAME_GERMANICUS imp = NICKNAME_THE_GERMAN ck3 = nick_the_german }
-#link = { imp = NICKNAME_RHAETICUS imp = NICKNAME_THE_RHAETIAN ck3 = ??? } #"The Rhetian"
-#link = { imp = NICKNAME_ITALICUS imp = NICKNAME_THE_ITALIOTE ck3 = ??? } #'The Italiote'
-#link = { imp = NICKNAME_ILLYRICUS imp = NICKNAME_THE_ILLYRIAN ck3 = ??? } #'The Illyrian'
+link = { imp = NICKNAME_RHAETICUS imp = NICKNAME_THE_RHAETIAN ck3 = nick_the_swiss }
```

Comment:
Why not keep it as "the Rhaetian"?

Reply:
Solved through localization. nick_the_swiss, nick_the_romanian, nick_the_russian actually convert to the Rhaetian, the Dacian, the Sarmatian.